test/integration/policyConsumptionTest.ts

```diff
@@ -2,24 +2,25 @@ import { strict as assert } from "assert";
 
 import { newTestUser } from "./clientHelper";
 import { Mjolnir } from "../../src/Mjolnir";
-import config from "../../src/config";
+import { read as configRead } from "../../src/config";
```

Comment:
to prevent this from happening again, the test should be added to the end of `tsconfig.json`.